Doc/library/shlex.rst

```diff
@@ -43,15 +43,16 @@ The :mod:`shlex` module defines the following functions:
    string that can safely be used as one token in a shell command line, for
    cases where you cannot use a list.
 
-   This idiom would be unsafe::
+   This idiom would be unsafe:
```

Comment:
Hello @berkerpeksag, only this point is still open.  Here I prefere to execute the test, because this example defines ``remote_command`` , that is used in a subsequent example.  In this way all tests pass.  If you want us to skip this test, we have to skip the subsequent tests that use ``remote_command`` name.  Let me know which option you prefere.  Thank you very much for your time :-)